<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>1261</width>
     <height>801</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="live">
    <attribute name="title">
     <string>live</string>
    </attribute>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>580</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imgLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>461</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>580</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>490</y>
       <width>491</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="imgLabel2">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>0</y>
       <width>611</width>
       <height>461</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>580</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>580</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>580</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>580</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="log">
    <attribute name="title">
     <string>log</string>
    </attribute>
    <widget class="QPushButton" name="load">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="toTime">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2021</year>
       <month>12</month>
       <day>18</day>
      </datetime>
     </property>
     <property name="time">
      <time>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
      </time>
     </property>
     <property name="displayFormat">
      <string>dd-MM-yy hh:mm </string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTableWidget" name="table">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>701</width>
       <height>641</height>
      </rect>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Time</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Mask</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Temp</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Photo</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fine</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Calibri</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>From</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="fromTime">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2020</year>
       <month>11</month>
       <day>16</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>dd-MM-yy hh:mm </string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
